dockers/docker-lldp-sv2/supervisord.conf.j2

```diff
@@ -31,7 +31,11 @@ stderr_logfile=syslog
 # - `-dd` means to stay in foreground, log warnings to console
 # - `-ddd` means to stay in foreground, log warnings and info to console
 # - `-dddd` means to stay in foreground, log all to console
+{% if DEVICE_METADATA['localhost']['sub_role'] is defined and DEVICE_METADATA['localhost']['sub_role']|length %}
+command=/usr/sbin/lldpd -d -I Ethernet* -C Ethernet*
+{% else %}
 command=/usr/sbin/lldpd -d -I Ethernet*,eth0 -C eth0
```

Comment:
what about rewritong this command line using sed,inside of supervisord.conf? And don't use jinja2 template for that?
We can change supervisor config in start.sh

Reply:
i suggest to make this code generic, detect if there is eth0, if there is then use eth0, if there is no, do not add eth0 into the command line. it is better than templating based on the sub_role.

Reply:
@pavel-shirshov : I tried couple of things to avoid jinja2 template 
a) supervisor environment variable 
b) sed command 
However both does not seems to work. supervisos.conf gets loaded whens supervisor comes up and seems there is some dependency we can't change run time from start.sh

@Iguohan:  eth0 is present both in container and host.  So we cannot distinguish that, Do we need to check if container is using --net= host or not ?

Reply:
@pavel-shirshov , what is the concern for using jinja2 template for supervisord.conf? if we can first generate and start supervisord, then we do not need to reload.

Reply:
@pavel-shirshov Based on above comment will be using jinja template for suerpvisor.conf

Reply:
@lguohan It looks like we're going to have another snowflake in SONiC. Everywhere we start with "start.sh" script. Here the "start.sh" is not run on the start (So the name is misleading). And we have another start script, with another name. This is confusing.